roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -363,3 +363,5 @@ etcd_events_peer_addresses: |-
   {% for item in groups['etcd'] -%}
     {{ "etcd"+loop.index|string }}-events=https://{{ hostvars[item].access_ip | default(hostvars[item].ip | default(hostvars[item].ansible_default_ipv4['address'])) }}:2382{% if not loop.last %},{% endif %}
   {%- endfor %}
+
+podsecuritypolicy_enabled: "{% if rbac_enabled %}{% if kube_version | version_compare('v1.10', '<') %}{{ 'PodSecurityPolicy' in kube_apiserver_admission_control|default([]) }}{% else %}{{ 'PodSecurityPolicy' in kube_apiserver_enable_admission_plugins|default([]) }}{% endif %}{% else %}false{% endif %}"
```

Comment:
could you indent in multiple lines all conditions?  this is hard to follow and read.

Reply:
@ant31 After all, I changed my mind make `podsecuritypolicy_enabled` the trigger for pod security policy enabling instead of relying on the presence of the admission controller in the list (it seems to be the way things are done in kubespray like auditing). 
So problem solved I guess